src/nix/flake.cc

```diff
@@ -1343,6 +1343,39 @@ struct CmdFlakePrefetch : FlakeCommand, MixJSON
     }
 };
 
+struct CmdFlakeRoot : FlakeCommand
+{
+    bool asRef = true;
+    CmdFlakeRoot()
+    {
+        addFlag({
+            .longName = "as-ref",
+            .shortName = 'r',
+            .description = "Show flake root as a normalized flake ref.",
+            .handler = {&asRef, false}
+        });
+    }
+
+    std::string description() override
+    {
+        return "get flake root";
```

Comment:
Should this be something like "get the root directory of a flake"? Not entirely clear what "flake root" means.

Reply:
changed the description